### ISSUE TITLE
Request - Quads Question

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#9848]
Help us understand your request (check below):

- [ ] search or data request (help with SQL or provide the specific fields you are trying to retrieve for your collection)
- [ ] add a new Github user to the Arctos Users team
- [ ] a new barcode series
- [x] a new term for data entry or management 
- [ ] a change or new report or label
- [ ] update to an Agent (split, merge, or other type of change)
- [ ] bulkload changes (to Agents, Identifiers, etc that are not already available as a bulkloading tool)
- [ ] new feature or otherwise a good idea to make Arctos even better
- [ ] quarantine taxon name and suggested replacement (please supply links)
- [ ] API key request (see https://handbook.arctosdb.org/documentation/api.html for requirements)
- [ ] error or bug (please include enough specific information to reproduce)
- [ ] general inquiry (or email arctoscontact@googlegroups.com)
- [ ] access issue (see https://arctosdb.org/faq/)
- [ ] other

**Describe what you're trying to do** 

I have around 400 quads to add to the code table for UCM. While working on this, I found a csv list of all the USGS maps with their metadata. My question is - can we just use this list? Maps are repeated as they have been issued in multiple editions, but we can name them using state, year and scale so that each has a unique value. If we don't want to just add them all, can we use this to populate the names we need and make all of the descriptions for values currently in the table consistent?

Before I start working on a code table request, I'd prefer to get a plan in place.

The list came from here - https://prd-tnm.s3.amazonaws.com/index.html?prefix=StagedProducts/Maps/Metadata/
which I reached using a link from https://www.usgs.gov/programs/national-geospatial-program/topographic-maps

I've uploaded a csv to the Google Drive - https://drive.google.com/file/d/1HUzEfrXqQMjR2CpKblDogxOQ1lcKsHG2/view?usp=drive_link 

**How-To**

------------

Please review the [Arctos Code of Conduct](https://arctosdb.org/code-of-conduct/) to ensure respectful communication, and see <https://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html> for more information about submitting and managing Issues.

**Comment by dustymc** [comment]
I don't have time to dig very deep right now but: Any source of bigger-than-Arctos standardization kinda always sounds like a good idea, if something functional can be worked out (quick glance looks promising...) then this sounds great.

See also https://handbook.arctosdb.org/documentation/authorities.html#ctquad

**Comment by mkoo** [comment]
Discussed in data migration meeting. Let's use this to update the CT (https://arctos.database.museum/info/ctDocumentation.cfm?table=ctquad)-- 
Current plan: use the existing ct, bulkload topoquad+scale values if it's missing from table. 
Update the existing ones that lack description (DO NOT USE ones) with data from CSV. 
I'll prep files for dusty.
If we need to prioritize batches, do the western states first.
In this first pass, I am skipping AK quads since they have already been added and most to the default scale of 1:250000

Question: do we want to add the footprint (WKT)? we dont need to now but know it exists in the USGS export

I also will draft a better table definition
